### ISSUE TITLE
Starting tail process after recieving http request

### ISSUE THREAD

**Issue by mikossheev** [mthenw/frontail#132]
Here is my case:
Frontail is used to monitor a bunch of folders with log files (one process per file, to view them separately).
There is a script that searches for the log files in directory, launches frontail and adds an nginx location for it.
Sometimes the quantity of processes getting to big, and recources are not used efficently. 

Here is a question:
Is there a way to modify frontail for it to launch its own process, but the tail process to be launched only when there is a request to frontail?

**Comment by ssbarnea** [comment]
If I am not wrong what you are are looking is more of what I described in https://medium.com/@sbarnea/how-to-share-terminal-output-on-the-web-83bf73c05c4c article, where the closes solution I found was seashells. 

I came to frontail while trying to find other alternatives but apparently frontail lack few critical feature like web server mode that could be used to serve "recordings" even after client disconnected, ability to auto-generate sharable urls and so on.

I am curious if @mthenw is looking to implement features like these in the future.
